src/test/java/com/traveltime/sdk/TimeMapTest.java

```diff
@@ -164,7 +166,9 @@ private List<DepartureSearch> createDepartureSearch(Coordinates coords, Transpor
             transportation,
             Instant.now(),
             900,
-            new Range(true, 400)
+            new Range(true, 400),
+            false,
+            false
```

Comment:
maybe there is some default values that could be set?
Since this would become a braking change if I understand correctly

Reply:
Its just the tests, if you use the builder you dont need to set each value.